modules/mco-update-boot-images-disable.adoc

```diff
@@ -9,6 +9,21 @@
 
 To disable the updated boot image feature, edit the `MachineConfiguration` object so that the `machineManagers` field is an empty array.
 
+[NOTE]
+====
+If you are upgrading a Google Cloud Platform (GCP) or Amazon Web Services (AWS) cluster from {product-title} 4.18 to 4.19, and you have not configured the `managedBootImages` parameter, the upgrade is blocked with the _This cluster is GCP or AWS but lacks a boot image configuration._ message. The update is blocked intentionally in order to alert you that the default updated boot image behavior is changing between version 4.18 and 4.19 to enable updated boot images by default.
```

Comment:
Instead of 4.19, would a later release be better? I'm more thinking in terms if this doc update is published before 4.19 is release, some kickback might come our way.

Reply:
Do attributes exist for GCP and AWS? Do they for OCP core.

Reply:
```suggestion
If you are upgrading a Google Cloud Platform (GCP) or an Amazon Web Services (AWS) cluster from {product-title} {product-version} to 4.19, and you have not configured the `managedBootImages` parameter, the upgrade is blocked and a `This cluster is GCP or AWS but lacks a boot image configuration` message is outputted. The update is blocked intentionally to alert you that the default updated boot image behavior is changing between version {product-version} and 4.19 to enable updated boot images by default.
```

Reply:
>  I'm more thinking in terms if this doc update is published before 4.19 is release

Yes, good point. I can wait on the merge until 4.19 is released.